### PR TITLE
Added webhook cleanup step to upgrade procedure

### PR DESCRIPTION
**Pull request by zimnx**

### CODE REVIEW

File: docs/source/upgrade.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ Sidecar image is going to be upgraded automatically, so a rolling restart of you
     ```
     kubectl delete namespace scylla-operator-system --wait=true
     ```
+1. Remove old webhooks:
+    ```
+    kubectl delete MutatingWebhookConfiguration scylla-operator-mutating-webhook-configuration
+    kubectl delete ValidatingWebhookConfiguration scylla-operator-validating-webhook-configuration
```
Comment (tnozicka) [inline docs/source/upgrade.md:23]: We still have a validating webhook, right?

Comment (zimnx) [reply docs/source/upgrade.md:23]: We do, but we use different name for the object.

Comment (tnozicka) [reply docs/source/upgrade.md:23]: you are right, I didn't check the name

**Review by tnozicka — DISMISSED** [review]
/lgtm

**Review by tnozicka — APPROVED** [review]

**Pull request merged**